waspc/cli/Wasp/Cli/Command/Db/Migrate.hs

```diff
@@ -34,69 +28,9 @@ migrateDev = do
           </> Cli.Common.dotWaspDirInWaspProjectDir
           </> Cli.Common.generatedCodeDirInDotWaspDir
 
-  -- TODO(matija): It might make sense that this (copying migrations folder from source to
-  -- the generated proejct) is responsibility of the generator. Since migrations can also be
-  -- considered part of a "source" code, then generator could take care of it and this command
-  -- wouldn't have to deal with it. We opened an issue on Github about this.
-  --
-  -- NOTE(matija): we need to copy migrations down before running "migrate dev" to make sure
-  -- all the latest migrations are in the generated project (e.g. Wasp dev checked out something
-  -- new) - otherwise "dev" would create a new migration for that and we would end up with two
-  -- migrations doing the same thing (which might result in conflict, e.g. during db creation).
```

Comment:
Compile is already called before we migrate, so this happens automatically now too (see `Wasp.Cli.Command.Db.makeDbCommand`).

Reply:
Yeah sure I can add, prob good to be explicit. 👍🏻

Reply:
Updated the comment like so to mention the previous compile (in this commit https://github.com/wasp-lang/wasp/pull/420/commits/42a03534386d45f19aed9889088fb3f05ccf9f85):
```haskell
-- | NOTE(shayne): Performs database schema migration (based on current schema) in the generated project.
-- This assumes the wasp project migrations dir was copied by a previous compile.
-- The migrate function takes care of copying migrations from the generated project back to the source code.
migrateDev :: Command ()
```